test/test-client.js

```diff
@@ -335,4 +335,94 @@ describe('Imgix client:', function describeSuite() {
       assert.equal(expectation, result);
     });
   });
+
+  describe('Calling buildSrcSet()', function describeSuite() {
+    describe('width descriptor mode', function describeSuite() {
```

Comment:
Rather than testing this explicitly as "width descriptor mode" and "dpr mode", can we instead test that it behaves correctly under the actual-use scenarios?

- With a `w` parameter provided
- With a `h` parameter provided
- With a `w` and an `h` parameter provided
- ... et cetera